src/Control/Monad/Aff.js

```diff
@@ -96,6 +96,26 @@ exports._forkAff = function (nonCanceler, aff) {
   };
 }
 
+exports._forkAll = function (nonCanceler, foldl, affs) {
+  var voidF = function(){};
+
+  return function(success, error) {
+    foldl(function(_) {
+      return function(aff) {
+        aff(voidF, voidF);
+      };
+    })({})(affs);
+
+    try {
+      success({});
+    } catch(e) {
+      error(e);
+    }
+
+    return nonCanceler;
```

Comment:
I get what the canceller does, and yes, it's very similar to what `cancelWith` does. Should this be part of the primary canceller for this action, or should it push this canceller to `success` like in `forkAff`?